src/pages/AddNewCake.js

```diff
@@ -1,32 +1,38 @@
 import "./AddNewCake.css";
+import InputRecipe from "../components/inputComponents/inputRecipe";
+import InputIngredients from "../components/inputComponents/inputIngredients";
+import ImageInput from "../components/inputComponents/imageInput";
+
+import InputItemName from "../components/inputComponents/InputItemName";
 
 export default function AddNewCake() {
+  function handleSubmit(event) {
+    event.preventDefault(event);
+    const form = event.target;
+    const stepsToRecipe = form.inputRecipe.value;
+    const ingredients = form.inputIngredients.value;
+    const itemName = form.itemName.value;
+
+    const recipeData = {
+      itemName,
+      stepsToRecipe,
+      ingredients,
+    };
```

Comment:
You should add an id to every recipe you create. You could use the library `uuidv4` https://www.npmjs.com/package/uuidv4 for that.
this will allow to have unique ids for each recipe you create, and also to use that id as a react key for instance.

John, Benedikt, and I think many others have done it already, you could ask them as well

Reply:
Thanks! I was planning to do it sooner or later. :)